ruby_test.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class TestHound
+  NAME = "Wormbots"
+  VERSION = '0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.